src/berrybrew.cs

```diff
@@ -181,7 +221,7 @@ internal static void Config()
 
         internal static string Version()
         {
-            return "0.12.1.20160302";
+            return "0.12.1.2016040101";
```

Comment:
After I take more deeper look to the changes, I will change it to 0.12.2.20160400. Last two digits are for iterations, not day of month. For 100 iterations per month are *more than plenty*. 100 of them per day are way, way excessive. Date string is to ease correlation with latest Strawberry releases.